torch/nested/_internal/nested_int.py

```diff
@@ -34,14 +36,32 @@ def _ge(lhs: Any, rhs: Any) -> bool:
         raise ValueError("inputs unsupported")
 
 
+def _get_tensor_id(t: torch.Tensor) -> int:
+    ret: List[Optional[int]] = [None]
```

Comment:
I don't get why this is a List :p

Reply:
I need to be able to mutate it in the inner scope, and non-local didn't seem to work for me here